getting-started/download.md

```diff
@@ -1,43 +1,70 @@
-| [WolvenKit Nightly](https://github.com/WolvenKit/WolvenKit-nightly-releases/) | [![GitHub release (latest by date)](https://img.shields.io/github/v/release/WolvenKit/WolvenKit-nightly-releases)](https://github.com/WolvenKit/WolvenKit-nightly-releases/releases/latest) | ![GitHub Workflow Status](https://img.shields.io/github/workflow/status/WolvenKit/WolvenKit/WolvenKit-Nightly) |
-| [WolvenKit](https://github.com/WolvenKit/WolvenKit/)                          | [![GitHub release (latest by date)](https://img.shields.io/github/v/release/WolvenKit/WolvenKit)](https://github.com/WolvenKit/WolvenKit/releases/latest)                                   | ![GitHub branch checks state](https://img.shields.io/github/workflow/status/WolvenKit/WolvenKit/check-only)    |
-
-The latest GitHub WolvenKit release will be marked on the left column with green text. Expand the **Assets** tab to view available downloads for any given WolvenKit release.
+The recommended download is the WolvenKit Nightly installer
+
+<table>
+  <tr align="left">
+    <th>Source Code</th>
+    <th>Installer</th>
+    <th>Checks</th>
+  </tr>
+  <tr>
+    <td>
+      <a href="https://github.com/WolvenKit/WolvenKit-nightly-releases/">
+        WolvenKit Nightly
+      </a>
+    </td>
+    <td>
+      <a href="https://github.com/WolvenKit/WolvenKit-nightly-releases/">
+        <img src="https://img.shields.io/github/v/release/WolvenKit/WolvenKit-nightly-releases" alt="GitHub nightly release (latest by date)">
+      </a>
+    </td>
+    <td>
+      <img src="https://img.shields.io/github/workflow/status/WolvenKit/WolvenKit/WolvenKit-Nightly" alt="GitHub branch checks">
+    </td>
+  </tr>
+  <tr>
+    <td>
+      <a href="https://github.com/WolvenKit/WolvenKit/">
+        WolvenKit Stable
+      </a>
+    </td>
+    <td>
+      <a href="https://github.com/WolvenKit/WolvenKit/releases/latest">
+        <img src="https://img.shields.io/github/v/release/WolvenKit/WolvenKit" alt="GitHub stable release (latest by date)">
+      </a>
+    </td>
+    <td>
+      <img src="https://img.shields.io/github/workflow/status/WolvenKit/WolvenKit/check-only" alt="GitHub branch checks">
+    </td>
+  </tr>
+</table>
 
 {% embed url="https://www.nexusmods.com/cyberpunk2077/mods/2201" %}
 
-### WolvenKit
+## System Requirements
 
-The main WolvenKit application in a portable zip file (Recommended)
+* 64-bit Windows 10/11
+* .NET 6.0 (x64)
 
 ### WolvenKit Installer
 
-The main WolvenKit application in a Microsoft installer package
+The main WolvenKit application in a Microsoft installer package.<br />
+<a href="https://wiki.redmodding.org/wolvenkit/wolvenkit-app/home">User guides</a>
 
 ### WolvenKit Console
 
-The standalone command-line based community modkit for Cyberpunk (Originally CP77 Tools)
+The standalone command-line based community modkit that replaced CP77 Tools.<br />
```

Comment:
WolvenKIt CLI did not "replace" CP77 Tools; it is CP77 Tools.

Reply:
Cool, I can appreciate the author of CP77 Tools wanting their hard work carried on into WolvenKit.

Reply:
I am the author of CP77Tools :) but yeah I’m fact CP77Tools is still used as term so jato is technically correct